aws/resource_aws_security_group_rule.go

```diff
@@ -98,6 +98,22 @@ func resourceAwsSecurityGroupRule() *schema.Resource {
 				ForceNew:      true,
 				Computed:      true,
 				ConflictsWith: []string{"cidr_blocks", "self"},
+				// The following documentation https://docs.aws.amazon.com/AmazonVPC/latest/PeeringGuide/vpc-peering-security-groups.html
```

Comment:
This suppression function is built on the premise that all security groups have a unique ID. I couldn't find any explicit piece of documentation online that this was true, however the documentation for making cross account SG references does not prefix the reference with an account number... which to me means AWS ensures uniqueness of SG ids else things would be super broken/collisions galore in the wild.